package.json

```diff
@@ -71,7 +71,44 @@
         "title": "Cycle Repeat Mode",
         "category": "Youtube Music"
       }
-    ]
+    ],
+    "productIconThemes": [
+      {
+        "id": "material-ui-font-icons",
+        "label": "Material UI font Icons",
+        "path": "./productIcons/material-ui-font-icons.json"
+      }
+    ],
```

Comment:
I'm not sure if we need this. [The docs](https://code.visualstudio.com/api/extension-guides/product-icon-theme) suggest this is for specifying that we expose a theme which I don't think we do. Also `./productIcons/material-ui-font-icons.json` is not added by this PR and didn't already exist.
```suggestion
```

Reply:
good catch, it's a remnant from before defining the icons. I missed removing it before the final commit